### PR TITLE
Fix oauth login

### PR DESCRIPTION
**Pull request by verschoof**

### CODE REVIEW

File: src/Elewant/FrontendBundle/Entity/User.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ public function getRoles() : array
 
     public function getUsername() : string
     {
-        return '';
+        return $this->username();
```
Comment (ramondelafuente) [inline src/Elewant/FrontendBundle/Entity/User.php:132]: So when this change is merged, the oauth login works?

**Comment by ramondelafuente** [comment]
The added comment block does nothing to help us. The typehint as well as the return type are declared in code already. I propose we don't use comments like that at all.

File: src/Elewant/FrontendBundle/Entity/User.php (hunk 1 above)
Comment (verschoof) [reply src/Elewant/FrontendBundle/Entity/User.php:132]: yes...

**Comment by verschoof** [comment]
I add the comment to be consistent with the rest of the file.
So if we remove all the comment its fine by me :) 👍

**Pull request merged**